sdcm/sct_config.py

```diff
@@ -1821,15 +1859,15 @@ def __init__(self):  # noqa: PLR0912, PLR0914, PLR0915
 
         # 1) load the default backend config files
         files = anyconfig.load(list(backend_config_files))
-        anyconfig.merge(self, files)
+        merge_dicts_append_strings(self, files)
 
         # 2) load the config files
         try:
             for conf_file in list(config_files):
                 if not os.path.exists(conf_file):
                     raise FileNotFoundError(f"Couldn't find config file: {conf_file}")
             files = anyconfig.load(list(config_files))
-            anyconfig.merge(self, files)
+            merge_dicts_append_strings(self, files)
```

Comment:
Commit messages says that the `config_files` is exception. So, is it?

Reply:
the commit message says that about using `SCT_CONFIG_FILES='["++", "new_config.yaml"]'`
not that you can't use this feature inside config files, for example you can have two config files like that:

```yaml
# config1.yaml
append_scylla_args: '-a -b -c'
```

```yaml
# config2.yaml
append_scylla_args: '++ -d -e -f' 
```

and use them ontop of each other

some environment values are used before the configuration files, like SCT_CONFIG_FILE, SCT_REGION_NAME, SCT_CLUSTER_BACKEND, variable that holds versionss,
and using append on them is much more complicated, and doesn't make any sense (at least to me)